### PR TITLE
sf Nov 4 RNs (both stable and stable-rc)

### PR DESCRIPTION
**Pull request by jshackell-sfdc**

### CODE REVIEW

File: releasenotes/sf/README.md [hunk 1]
```diff
@@ -2,12 +2,24 @@
 
 Here are the new and changed features in recent updates of the `sf` executable of Salesforce CLI.
 
-We publish a new stable version of `sf` on Thursdays that will be installed automatically when you install or update `sfdx`.
+We publish a new stable version of `sf` on Thursdays; this version is installed automatically when you install or update `sfdx`. At the same time we also publish an `sf` release candidate that contains changes that we plan to include in next week's `sf` release.
```
Comment (RodEsp) [inline releasenotes/sf/README.md]:
```suggestion
We publish a new stable version of `sf` on Wednesday; this version is installed automatically when you install or update `sfdx`. At the same time we also publish an `sf` release candidate that contains changes that we plan to include in next week's `sf` release.
```

**Review by RodEsp — COMMENTED** [review]
A few fixes. 
* We promote `stable-rc` to `stable` on Wednesday for `sf`.
* The `sfdx update` bug was actually for 1.2.0.
* Please feel free to change up the wording as you see fit for the install stuff I put under 1.3.0 (stable-rc this week)

File: releasenotes/sf/README.md [hunk 2]
```diff
@@ -2,12 +2,24 @@
...
 
+## 1.3.0 (Nov 11, 2021) [stable-rc]
+
+These changes are in the Salesforce CLI (`sf` executable) release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.
+
+* FIX: We fixed a bug that prevented `sfdx update` from also updating `sf` to its latest version.
+
+## 1.2.0 (Nov 4, 2021)
+
+* FIX: We fixed some under-the-hood bugs.
```
Comment (RodEsp) [inline releasenotes/sf/README.md]:
```suggestion
## 1.3.0 (Nov 11, 2021) [stable-rc]

These changes are in the Salesforce CLI (`sf` executable) release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.

* FIX: Installation of `sf` will be much more reliable when installing `sfdx` via the installers and tarballs.

## 1.2.0 (Nov 4, 2021)

* FIX: We fixed a bug that prevented `sfdx update` from also updating `sf` to its latest version.
```

File: releasenotes/sf/README.md (hunk 1 above)
Comment (jshackell-sfdc) [reply releasenotes/sf/README.md]: Hmm, your change makes it sound like the new `sf` version will be available via `sfdx update` on Wednesday, which isn't true. (Right?)  That's why I said we publish it Thursday, even thought that's not 100% accurate.  But it IS accurate in that users have to wait until Thursday to get the new `sf` version via `sfdx update`.  Does that make sense?  I don't want to go into the details of how/when we promote, etc...

Comment (jshackell-sfdc) [reply releasenotes/sf/README.md]: Oh, I'm overthinking this.   Saying Wed is fine :)

**Pull request merged**